models/aeif_cond_alpha.h

```diff
@@ -84,51 +86,51 @@ This implementation uses the embedded 4th order Runge-Kutta-Fehlberg solver with
 adaptive step size to integrate the differential equation.
 
 The membrane potential is given by the following differential equation:
-C dV/dt= -g_L(V-E_L)+g_L*Delta_T*exp((V-V_T)/Delta_T)-g_e(t)(V-E_e)
-                                                     -g_i(t)(V-E_i)-w +I_e
+@f[ C dV/dt= -g_L(V-E_L)+g_L*Delta_T*exp((V-V_T)/Delta_T)-g_e(t)(V-E_e)
+                                                     -g_i(t)(V-E_i)-w +I_e @f]
 
 and
 
-tau_w * dw/dt= a(V-E_L) -W
+@f[ tau_w * dw/dt= a(V-E_L) -W @f]
 
 Parameters:
 
 The following parameters can be set in the status dictionary.
 
 Dynamic state variables:
-  V_m        double - Membrane potential in mV
-  g_ex       double - Excitatory synaptic conductance in nS.
-  dg_ex      double - First derivative of g_ex in nS/ms
-  g_in       double - Inhibitory synaptic conductance in nS.
-  dg_in      double - First derivative of g_in in nS/ms.
-  w          double - Spike-adaptation current in pA.
+-  V_m        double - Membrane potential in mV
```

Comment:
Any way to turn this into even nicer tables when rendered?

The `double` data type information here makes little sense. Since almost all users will be working in Python, if we give it at all it should be `float`.

Reply:
I think we can more easily address this when generating the parameter tables from NESTML. C.f. https://github.com/nest/nestml/pull/446 for the work-in-progress PR doing that.

Reply:
I've changed the lists into tables now for parameters - (https://nest-test.readthedocs.io/en/extract_models_breathe/models/neurons.html) how does that look? Should the double column be completely removed or replaced with float?